configure_git_repository.md

```diff
@@ -18,7 +18,7 @@ Please stick to it unless you have special needs.
   * Default branch: `develop`. Click *update*
   * Add these rules for the two branches `develop` and `master`:
     * Require pull request reviews before merging
-    * Require status checks to pass before merging
+    * Require status checks to pass before merging (after you configured the CI add it to the required checks)
```

Comment:
Ik, it's not the best spot but better than nothing, isn't it?

Reply:
Yes, what about mentioning something also in the CI/CD setup?